django_ilmoitin/apps.py

```diff
@@ -5,3 +5,6 @@
 class DjangoIlmoitinConfig(AppConfig):
     name = "django_ilmoitin"
     verbose_name = _("Ilmoitin")
+
+    def ready(self):
+        import django_ilmoitin.api.schema  # noqa
```

Comment:
Just out of curiosity, why is this import required here? I think the graphql types in this app would get automatically registered by just subclassing `django_ilmoitin.api.schema.Query` in the app's Query where `django_ilmoitin` package is used.

Reply:
In my mind it made sense to me as well, but without manually "loading" the schema package beforehand, I kept bumping into this error:
![image](https://user-images.githubusercontent.com/15201480/85988522-cfa55100-b9f7-11ea-8dd1-355a06fffdc4.png)

Reply:
ok, this import during `ready` is not actually required, if in the project you do `import django_ilmoitin.api` or something similar. resolving this thread.